proto/demo_grpc/simple_router_mgr.cpp

```diff
@@ -35,6 +35,9 @@
 
 #include "google/rpc/code.pb.h"
 
+namespace p4rt = ::p4::v1;
```

Comment:
I feel p4v1 is a better fit. First, its the same number of letters :) But more importantly, the current p4rt is de-emphasizing that these are v1 messages. In the future, when we have v2, we would need to introduce a new alias, and p4v2 would be better than something like p4rt2. Also, at that point, this p4rt would become ambiguous wrt which version it refers to. Finally, re-pointing p4rt to ::p4::v2 will be a no-no (IMHO) since they will be incompatible by definition.
I would say the same for config (p4configv1).
What do you think?

Reply:
If you are referring specifically to the demo client under demo_grpc, then I agree it should probably use `p4::v1` or `p4v1` instead of `p4rt`.
However if this is a generic remark that includes the server implementation files, I am not sure I agree. Actually the reason why I introduced the `p4rt` alias was to make potential future changes (including v1 -> v2) easier to implement. Even though this is far in the future, I would expect most messages to stay backward-compatible with the change from v1 to v2, and the namespace alias was meant to help keep the CL small.
That being said, there is no urgency to merge this (you also seem to agree with the .proto changes, which is the probably the most important thing to get right at the moment), so I can take some more time to think about this.

Reply:
I thought about this a little more, and looked at code which follows the guidelines where are trying to adhere to. I do style similar to yours (p4rt=::p4::v1) as well as variants (p4rtv1, p4rtv1, p4rt_v1, etc.). I guess there is no hard and fast rule around this. LGTM.